app/views/admin/poll/polls/_questions.html.erb

```diff
@@ -20,10 +20,6 @@
           </strong>
         </td>
         <td class="text-right">
```

Comment:
I think in this file you should delete this empty `<td class="text-right"></td>` and the unnecessary `<th>` on L12: `<th class="text-right"><%= t('admin.polls.show.table_assignment') %></th>` 😉